packages/react-router/modules/__tests__/Route-test.js

```diff
@@ -34,6 +34,23 @@ describe('A <Route>', () => {
 
     expect(node.innerHTML).toNotContain(TEXT)
   })
+
+  it('supports preact by nulling out children prop when empty array is passed', () => {
+    const TEXT = 'Mrs. Kato'
+    const node = document.createElement('div')
+
+    ReactDOM.render((
+      <MemoryRouter initialEntries={[ '/' ]}>
+        <Route path="/" render={() => (
+          <h1>{TEXT}</h1>
+        )}>
+          {[]}
```

Comment:
The way this test is written should trigger [this warning](https://github.com/ReactTraining/react-router/blob/b09476946b5d6fa670b23e7d3c6b62fda3a44017/packages/react-router/modules/Route.js#L31-L34), because you're using both the `<Route render>` and `<Route children>` props. Let's just get rid of the test and put a note in the code.